repository.rosinstall

```diff
@@ -48,3 +48,18 @@
     local-name: shadow_robot_ethercat
     uri: https://github.com/shadow-robot/sr-ros-interface-ethercat.git
     version: kinetic-devel
+
+- git:
+    local-name: warehouse_ros_mongo
+    uri: https://github.com/ros-planning/warehouse_ros_mongo.git
+    version: jade-devel
+
+- git:
+    local-name: warehouse_ros
+    uri: https://github.com/ros-planning/warehouse_ros.git
+    version: jade-devel
+
+- git:
```

Comment:
This is not a ROS package, so it should be installed through some other mechanism. (e.g. ansible role)